src/app/index.tsx

```diff
@@ -1,74 +1,33 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * Generated with the TypeScript template
- * https://github.com/react-native-community/react-native-template-typescript
- *
- * @format
- */
-
 import React from 'react';
 import {
   SafeAreaView,
   StyleSheet,
   ScrollView,
   View,
   Text,
-  StatusBar
+  StatusBar,
+  Image
 } from 'react-native';
 
-import {
-  Header,
-  LearnMoreLinks,
-  Colors,
-  DebugInstructions,
-  ReloadInstructions
-} from 'react-native/Libraries/NewAppScreen';
+import { Colors } from 'react-native/Libraries/NewAppScreen';
 
-declare const global: { HermesInternal: null | {} };
+import { ALICE_BLUE, CHARCOAL } from '../constants/colors';
 
 const App = () => {
   return (
     <>
       <StatusBar barStyle="dark-content" />
-      <SafeAreaView>
-        <ScrollView
-          contentInsetAdjustmentBehavior="automatic"
-          style={styles.scrollView}>
-          <Header />
-          {global.HermesInternal == null ? null : (
-            <View style={styles.engine}>
-              <Text style={styles.footer}>Engine: Hermes</Text>
-            </View>
-          )}
-          <View style={styles.body}>
-            <View style={styles.sectionContainer}>
-              <Text style={styles.sectionTitle}>Step One</Text>
-              <Text style={styles.sectionDescription}>
-                Edit <Text style={styles.highlight}>App.tsx</Text> to change
-                this screen and then come back to see your edits.
-              </Text>
-            </View>
-            <View style={styles.sectionContainer}>
-              <Text style={styles.sectionTitle}>See Your Changes</Text>
-              <Text style={styles.sectionDescription}>
-                <ReloadInstructions />
-              </Text>
-            </View>
-            <View style={styles.sectionContainer}>
-              <Text style={styles.sectionTitle}>Debug</Text>
-              <Text style={styles.sectionDescription}>
-                <DebugInstructions />
-              </Text>
+      <SafeAreaView style={styles.container}>
+        <ScrollView style={styles.scrollView}>
+          <View style={styles.bookContainer}>
+            <Image
+              style={styles.bookImg}
+              source={require('./assets/img_book1@3x.png.png')}
```

Comment:
We use another way to import assets:
```
import bookImage from './assets/img_book1.png';
...
source={bookImage}
...
```
Also, fix the asset name => `img_book1@3x.png`. Then download and add  `img_book1@2x.png` and`img_book1.png` from [Drive](https://drive.google.com/drive/u/0/folders/0B0bbTjmKxmYsdWRQX1hSd2ZRT1k)

As a convention, we define a `resizeMode` for the Image component => [Doc](https://reactnative.dev/docs/image#resizemode). For this case you can use `resizeMode={contain}`